moshi/src/main/java/com/squareup/moshi/-MoshiKotlinTypesExtensions.kt

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Does the name of this class show up in the bytecode?

Reply:
I’m tempted to request something else.

For some of the methods here . . . do the signatures interop with Moshi’s `Types` class? Can we rewrite that class in Kotlin and do the Kotlin conversion? That way we share the API for Java and Kotlin, rather than having one for each?

Reply:
If we rewrote it, we probably would need to make the stdlib a direct dependency rather than compileOnly. I'd be fine with just ditching this class entirely for now and save rewriting for a full rewrite later